home/tests/test_add_answer_profile.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
+
+
+@pytest.mark.django_db
+class TestModels:
+
+    def test_add_profile(self):
+        profile = self.get_profile()
+        assert profile.user.id >= 0  # check if user saved successfully and get an id
+        assert profile.gender == 'U'  # default gender is U
+        assert isinstance(profile, Profile)  # check if the profile is a instance of Profile
```

Comment:
Will these pass even if you remove the `profile.save()`?
It is unclear if the objects here are in sync with the db or not (I would expect a "fetch" operation if the intention is to cover the db as well).

In addition, I do not see a point in checking the type of profile, it is created as an instance of it explicitly on line 26.
Same for the other cases.

Reply:
> Will these pass even if you remove the `profile.save()?`
No, I check that the command success.

I added a tests on DB , I hope that's what you meant.